delivery_local_pickup/tests/test_website.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+import odoo.tests
+
+from .common import DeliveryLocalPickupCommon
+
+
+class TesDeliveryLOcalPickupWebsite(DeliveryLocalPickupCommon, odoo.tests.HttpCase):
+    def setUp(self):
+        super().setUp()
+
+    def test_local_pickup_tour(self):
+        self.picking.action_confirm()
+        self.picking.action_assign()
+        self.picking.send_to_shipper()
+        tour = "delivery_local_pickup_tour"
+        self.phantom_js(
+            "/local_pickup/%s" % self.picking.id,
+            "odoo.__DEBUG__.services['web_tour.tour'].run('%s')" % tour,
+            "odoo.__DEBUG__.services['web_tour.tour'].tours.%s.ready" % tour,
+        )
```

Comment:
In 13.0 exists this method that it's more readable. You can set too the user that runs the tour by setting `login="admin"` for example
```suggestion
        self.start_tour("/local_pickup/%s" % self.picking.id, "delivery_local_pickup_tour")
```